src/components/Wheel/index.js

```diff
@@ -1,36 +1,23 @@
 import React, { useRef, useEffect, useState } from 'react';
 
 import ModalWindow from '../ModalWindow';
-import { Button, CircularProgress } from '@material-ui/core';
+import { Button } from '@material-ui/core';
 import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
 
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 
-import { fetchFortuneData } from '../../actions/fortune';
+import { fetchFirebaseData } from '../../actions/firebase';
 
-const mapStateToProps = (props) => ({
-  fortune: props.fortune.fortune,
-  common: props.common,
-});
-
-const mapDispatchToProps = (dispatch) =>
-  bindActionCreators(
-    {
-      fetchFortuneData: fetchFortuneData,
-    },
-    dispatch,
-  );
+import FirebaseWrapper from '../../containers/FirebaseWrapper';
 
 const Wheel = (props) => {
-  const {
-    fortune = [],
-    common: { init = false, error = null },
-  } = props;
+  const { fortune = [] } = props;
 
-  if (init) {
-    props.fetchFortuneData();
-  }
+  useEffect(() => {
+    props.fetchFirebaseData();
```

Comment:
Why do you need to fetch firebase data here?

Reply:
You're right, I don't need to use fetchFirebaseData here. Updated in last commit